protostar/cli/command_test.py

```diff
@@ -19,3 +19,10 @@ def test_directory_type(tmpdir):
 def test_directory_type_checks_existence():
     with pytest.raises(AssertionError):
         Command.Argument.Type.directory("!@#$")
+
+
+def test_int_type():
+    ints = ["123", "0xf", "0b001", "0o444", "0xDEADC0DE"]
+    for i in ints:
+        result = Command.Argument.Type.int(i)
+        assert isinstance(result, int)
```

Comment:
It could be a parametrized test. You could also check if all representation are converted to the same value.

Reply:
could we also convert one e2e test to use this feature?